File: data/imported/knowledge/slides/java/merged/snippets/snippet_092.java
@Test
    void testCloneSlide() {
        try (var pres = new Presentation()) {
            ISlide slide = pres.getSlides().get(0);
            slide.getShapes().addAutoShape(ShapeType.RECTANGLE, 50, 50, 200, 100);
            pres.getSlides().addClone(slide);
            assertThat(pres.getSlides().size()).isEqualTo(2);
            assertThat(pres.getSlides().get(1).getShapes().size()).isGreaterThanOrEqualTo(1);
        }
    }

File: data/imported/knowledge/slides/cpp/merged/snippets/snippet_092.cpp
TEST_F(ThreeDFormatIntegrationTest, BevelTop) {
    Presentation pres;
    auto& slide = clear_slide(pres);
    auto& shape = slide.shapes().add_auto_shape(ShapeType::RECTANGLE, 100, 100, 200, 100);
    auto& tdf = shape.three_d_format();
    tdf.bevel_top().set_bevel_type(BevelPresetType::CIRCLE);
    tdf.bevel_top().set_width(10);
    tdf.bevel_top().set_height(5);

    auto pres2 = save_and_reopen(pres);
    auto& bt = pres2.slides()[0].shapes()[0].three_d_format().bevel_top();
    EXPECT_EQ(bt.bevel_type(), BevelPresetType::CIRCLE);
    EXPECT_EQ(bt.width(), 10);
    EXPECT_EQ(bt.height(), 5);
}